release-strategy.json

```diff
@@ -8,5 +8,5 @@
     "rawls":            {"dev_only": false},
     "sam":              {"dev_only": false},
     "buffer":           {"dev_only": false},
-    "workspacemanager": {"dev_only": true}
+    "workspacemanager": {"dev_only": false}
```

Comment:
so just to confirm - this wouldn't affect prod because prod is currently not enabled?

Reply:
yep